locales/en.yml

```diff
@@ -152,12 +173,27 @@ en:
           Only sccli services are supported. For example:
             docker, openshift and kubernetes
 
+      install_cli:
+        message: |-
+          # Binary %{when} available at %{path}
+          # run binary as:
+          # %{binary} <command>
+          export PATH=%{dir}:$PATH
+
+          # run following command to configure your shell:
+          # eval "$(VAGRANT_NO_COLOR=1 vagrant service-manager install-cli %{service} | tr -d '\r')"
+        unsupported_box: |-
+          Sorry, client binaries for CDK cannot be installed via the 'install-cli' command yet.
+          Refer to download the libraries yourself.
```

Comment:
i think we can do better than that. Let's put the actual link there or at least refer to access.redhat.com. I believe they are somewhere there